content/200-concepts/100-components/01-prisma-schema/06-relations/100-one-to-one-relations.mdx

```diff
@@ -43,12 +49,55 @@ model Profile {
 </tab>
 </TabbedContent>
 
-The `userId` relation scalar is a direct representation of the foreign key in the underlying database.This 1-1-relation expresses the following:
+The `userId` relation scalar is a direct representation of the foreign key in the underlying database. This one-to-one relation expresses the following:
```

Comment:
```suggestion
The `userId` relation scalar is a direct representation of the foreign key in the underlying database. This one-to-one relation expresses the following:
```
```suggestion
The `userId` relation scalar is a direct representation of the foreign key in the underlying database. This one-to-one relation expresses the following:
```

In light of our team discussion earlier, consider adding a link to the definition of scalar (data model page) on the first mention of the word on this page.